### PR TITLE
Added Dravit Lochan as a mentor in mentor.yml and img/mentors

### PR DESCRIPTION
**Pull request by Vishnu2875**
- [x] Included a Preview link and screenshot showing after and before the changes.
- [x] Included a description of the change below.
- [ ] Squashed the commits.

# Changes done in this Pull Request

- [Preview Link](https://vishnu2875.github.io/gci18.fossasia.org/)

- Fixes #630

## Description / Changes
Added Dravit Lochan as a mentor in mentor.yml and img/mentors

### After:
after adding dravit lochan as a mentor

- - - - - - - - - - - -

### CODE REVIEW

**Comment by Vishnu2875** [comment]
Done check once

**Comment by KSNINJA** [comment]
Squash your commits .

**Comment by Vishnu2875** [comment]
i cant squash my commits in my pc

**Comment by Vishnu2875** [comment]
i cant squash my commits in my old pc

**Comment by Vishnu2875** [comment]
can you please merge

File: _data/mentors.yml [hunk 1]
```diff
@@ -10,6 +10,11 @@
 #   lat: Mentor's Latitude
 #   lng: Mentor's Longitude
 
+- name: Dravit Lochan
+  github: DravitLochan
+  image: Dravit Lochan.jpg
+
+
```
Comment (sk9331657) [inline _data/mentors.yml]: @Vishnu2875  Remove this extra line else LGTM 👍 .

Comment (Vishnu2875) [reply _data/mentors.yml]: Done

**Comment by abishekvashok** [comment]
Thanks

**Pull request merged**